### PR TITLE
Added a new method to get the configuration information of the cluste…

### PR DESCRIPTION
**Pull request by v-jiezhang3**
…r from the new API

What does this implement/fix? Explain your changes.
---------------------------------------------------
…

Does this close any currently open issues?
------------------------------------------

Any relevant logs, screenshots, error output, etc.?
-------------------------------------

Any other comments?
-------------------
…

Has this been tested?
---------------------------
- [ ] Tested

### CODE REVIEW

**Review by WX59991 — DISMISSED** [review]

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/cluster/HDInsightNewAPI/ClusterOperationNewAPIImpl.java [hunk 1]
```diff
@@ -69,11 +69,22 @@ private Observable<ClusterConfiguration> getClusterConfigurationRequest(
                 .withUuidUserAgent()
                 .post(url, entity, null, null, ClusterConfiguration.class);
     }
+    private Observable<ClusterConfiguration> getClusterConfigurationRequestFormNewAPI(
```
Comment (Flanker32) [inline Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/cluster/HDInsightNewAPI/ClusterOperationNewAPIImpl.java]: please add empty line before methods

Comment (Flanker32) [inline Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/cluster/HDInsightNewAPI/ClusterOperationNewAPIImpl.java]: Could we just replace the implementation of `getClusterConfigurationRequest`, or we need to keep both the two methods for different cases? 
If we need to keep both of the methods, shall we rename the method to `getClusterConfigurationFromManagementEndpoint` and `getClusterConfigurationFromResourceManagerEndpoint`? Or we could extract a method and pass endpoint as a parameter

Comment (v-jiezhang3) [reply Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/cluster/HDInsightNewAPI/ClusterOperationNewAPIImpl.java]: got it

Comment (v-jiezhang3) [reply Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/cluster/HDInsightNewAPI/ClusterOperationNewAPIImpl.java]: got it

**Review by xiawangMs — APPROVED** [review]

**Pull request merged**